I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 432, in script
    Prepod "Здравствуйте, первокурсники, добро пожаловать в лучший институт нашего ВУЗа."
Exception: Sayer 'Prepod' is not defined.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 432, in script
    Prepod "Здравствуйте, первокурсники, добро пожаловать в лучший институт нашего ВУЗа."
  File "E:\новела\renpy-8.1.3-sdk\renpy\ast.py", line 890, in execute
    who = eval_who(self.who, self.who_fast)
  File "E:\новела\renpy-8.1.3-sdk\renpy\ast.py", line 783, in eval_who
    raise Exception("Sayer '%s' is not defined." % who)
Exception: Sayer 'Prepod' is not defined.

Windows-10-10.0.22621 AMD64
Ren'Py 8.1.3.23091805
Путь Программиста 1.0
Tue Oct 10 13:26:32 2023
